GitVersionConfig.yaml

```diff
@@ -1,3 +0,0 @@
-mode: ContinuousDeployment
-assembly-versioning-scheme: MajorMinorPatch 
-#next-version: 1.0.0
```

Comment:
I just did some quick reading on GitVersion. It seems this yml file I removed is related to it. I just don't see anywhere it was used in the build. And in a local build it certainly wasn't used to make all the version stamps agree consistently.